docs/visual-basic/programming-guide/concepts/linq/advanced-linq-to-xml-programming.md

```diff
@@ -1,27 +1,28 @@
 ---
```

Comment:
we're in the process of combining the LINQ to XML into the standard folder with issue #4728. I'd avoid touching articles in that section since all it's going to be doing is creating merge conflicts.

Reply:
Is there someone working currently on that issue? @mairaw

Reply:
yes, our vendor team

Reply:
Let me connect the PRs to that issue, but it's going very slowly.

Reply:
Ok. I'm going to close it for now. And after they're moved to standard, I'll look if they need updates.

Reply:
Sounds good. You had some good changes in here!